agent/containers/images/Makefile

```diff
@@ -27,22 +27,11 @@ IMAGE_REPO = docker://quay.io/pbench
 # Not intended to be overridden with an environment variable.
 _REPO_TEMPLATE = ../../ansible/pbench/agent/roles/pbench_repo_install/templates/etc/yum.repos.d/pbench.repo.j2
 
-# NOTE: Currently we require 5.2.2 of the PCP RPMs because using the 5.2.3
-# version prevents us from integrating with Grafana, see PCP Issue #1183,
-# https://github.com/performancecopilot/pcp/issues/1183.
-# NOTE: We also have to enumerate so many RPMs because for CentOS 7 the RPM
-# dependency resolver does not properly resolve to the same version RPMs. Once
-# we no longer have to use v5.2.2, we can just list 3 RPMs: pcp-zeroconf,
-# pcp-system-tools, and pcp-gui.
+# This list of PCP RPMs we need.
```

Comment:
It would be nice to replace the old comment, which provided some context for the URL/repo references, with something that's updated to provide your new references "for those who come after".